docs/optimization_guide/nncf_introduction.md

```diff
@@ -0,0 +1,63 @@
+# Neural Network Compression Framework {#docs_nncf_introduction}
+This document describes the Neural Network Compression Framework (NNCF) which is being developed as a separate project outside of OpenVINO&trade; but it is highly aligned with OpenVINO&trade; in terms of the supported optimization features and models. It is open-sourced and available on [GitHub](https://github.com/openvinotoolkit/nncf).
+
+## Introduction
+ Neural Network Compression Framework (NNCF) is aimed at optimizing Deep Neural Network (DNN) by applying optimization methods, such as quantization, pruning, etc., to the original framework model. It mostly provides in-training optimization capabilities which means that optimization methods require model fine-tuning during and after optimization. The diagram below shows the model optimization workflow using NNCF.
+ ![](../img/nncf_workflow.png)
+
+ ### Features
+ - Support optimization of PyTorch and TensorFlow 2.x models.
+ - Support of various optimization algorithms, applied during a model fine-tuning process to achieve a better performance-accuracy trade-off:
```

Comment:
Support of various optimization algorithms, applied during a model fine-tuning process, to achieve a better performance-accuracy trade-off